src/bot.py

```diff
@@ -125,9 +125,6 @@ def tweet_loop(self, api):
                     logging.info("Tweeted out a new flower bed!")
                     logging.info("The next tweet is scheduled to be made in {} minutes".format(cooldown))
                     time.sleep(cooldown)
-            except tweepy.RateLimitError:
-                logging.critical("Tweeting failed due to ratelimit. Waiting {} more minutes.".format(cooldown))
-                time.sleep(cooldown)

```

Comment:
Why did you remove both handlers? There should be at least one that halts the bot before it carries on.

Reply:
ok. So the main will contain only the TweepError exception, and the bot will contain RateLimit error and a generic exception handler.